pages/index.tsx

```diff
@@ -1,12 +1,14 @@
 import type { NextPage } from 'next'
 import HomeBackground from '../components/HomeBackground'
 import styles from '../styles/Home.module.css'
+import HomePageIdeaPicture  from '../components/HomePageIdeaPicture'
 
 const Home: NextPage = () => {
   return (
     <div className={styles.container}>
       <HomeBackground>
         {/* Inser here page */}
+      <HomePageIdeaPicture/>
```

Comment:
2 lines right, to be on es-lint criteria. 
Good reminder to setup lint in whole project 😄  thanks!